### PR TITLE
NFDIV 636 how do you want to apply page

### PR DESCRIPTION
**Pull request by lara-mpt**
### JIRA link ###
https://tools.hmcts.net/jira/browse/NFDIV-636

### Change description ###
Add the how do you want to apply page, functional test, and applicationType fields

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/steps/how-do-you-want-to-apply/template.njk [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      {% for item in helpPayingWhen %}
+        <li>{{ item | safe }}</li>
+      {% endfor %}
+    </ul>
+    <p>{{ helpText3 }}</p>
+  {% endset %}
+    {{ govukDetails({
+    summaryText: readMore,
+    html: summary
+  }) }}
```
Comment (tomxelliott) [inline src/main/steps/how-do-you-want-to-apply/template.njk:29]: Could this be formatted more elegantly like this?
`{{ 
    govukDetails({
        summaryText: readMore,
        html: summary
    }) 
}}`
Just a thought...

Comment (tomxelliott) [reply src/main/steps/how-do-you-want-to-apply/template.njk:29]: Was trying to put suggested formatting here but the comments code option doesn't seem to want to let me :(

Comment (tomxelliott) [reply src/main/steps/how-do-you-want-to-apply/template.njk:29]:
```
{{ 
    govukDetails({
        summaryText: readMore,
        html: summary
    })
}}
```

**Review by tomxelliott — APPROVED** [review]
Looks good to me. Just the minor comment re: formatting 👍

**Review by SebFlippence — APPROVED** [review]

**Pull request merged**